src/Presentation/Presenters/InternalErrorHtmlPresenter.php

```diff
@@ -12,16 +12,15 @@
  * @licence GNU GPL v2+
  * @author Kai Nissen < kai.nissen@wikimedia.de >
  */
-class InternalErrorHtmlPresenter {
+class InternalErrorHtmlPresenter implements ExceptionHtmlPresenterInterface {
 
-	private $template;
+	private TwigTemplate $template;
 
-	public function __construct( TwigTemplate $template ) {
+	public function setTemplate( TwigTemplate $template ): void {
 		$this->template = $template;
 	}
 
 	public function present( \Exception $exception ): string {
-		return $this->template->render( [ 'message' => $exception->getMessage() ] );
+		return $this->template->render( [ 'message' => false ] );
```

Comment:
You could just render without `message`, IMHO that's better than a `string|boolean` type.